### PR TITLE
(feat) submit-plan accepts --app input

### PR DESCRIPTION
**Pull request by tphoney**
part of https://github.com/overmindtech/deploy/issues/1062

### CODE REVIEW

File: end-change/action.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ inputs:
   log:
     description: The log level for the job
     default: info
+  app:
+    description: "The overmind instance to connect to. eg `https://app.overmind.tech`"
```
Comment (DavidS-ovm) [inline end-change/action.yml:19]:
```suggestion
    description: "The Overmind instance to connect to. defaults to `https://app.overmind.tech`"
```

File: submit-plan/action.yml [hunk 2]
```diff
@@ -106,6 +113,7 @@ runs:
             $tf_plan_output_arg \
             --log '${{ inputs.log }}' \
             ${{ inputs.plan-json }} \
+            $app_arg \
```
Comment (DavidS-ovm) [inline submit-plan/action.yml:116]: I don't think that the go args parser is so strict, but _I_ would prefer all options before all arguments.

File: start-change/action.yml [hunk 3]
```diff
@@ -15,6 +15,9 @@ inputs:
   log:
     description: The log level for the job
     default: info
+  app:
+    description: "The overmind instance to connect to. eg `https://app.overmind.tech`"
```
Comment (DavidS-ovm) [inline start-change/action.yml:19]:
```suggestion
    description: "The Overmind instance to connect to. defaults to `https://app.overmind.tech`"
```

File: submit-plan/action.yml [hunk 4]
```diff
@@ -29,6 +29,9 @@ inputs:
   number:
     description: "Pull request number to report results back."
     default: ${{ github.event.number }}
+  app:
+    description: "The overmind instance to connect to. eg `https://app.overmind.tech`"
```
Comment (DavidS-ovm) [inline submit-plan/action.yml:33]:
```suggestion
    description: "The Overmind instance to connect to. defaults to `https://app.overmind.tech`"
```

**Review by DavidS-ovm — APPROVED** [review]
with some notes

File: submit-plan/action.yml (hunk 2 above)
Comment (tphoney) [reply submit-plan/action.yml:115]: moved it before the inputs.plan-json

**Review by DavidS-ovm — APPROVED** [review]
:shipit:

**Pull request merged**